src/test/util/setup_common.cpp

```diff
@@ -76,6 +76,9 @@ const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
 
 constexpr inline auto TEST_DIR_PATH_ELEMENT{"test_common bitcoin"}; // Includes a space to catch possible path escape issues.
 
+/* Used to group tests executed within the same binary run under the same directory */
+static const std::string g_init_time{util::ToString(TicksSinceEpoch<std::chrono::nanoseconds>(SystemClock::now()))};
+
```

Comment:
> Replicating the functional test framework behavior.

Should we use a prettier format here just like done in the functional tests? With this change the directory name is in plain nanoseconds like below.

```
drwx------    3 rkrux  staff    96B Nov 15 11:09 1731649167593349000
drwx------    3 rkrux  staff    96B Nov 15 11:11 1731649280506089000
```

Whereas for functional tests, the name is a little easier to read by using a date time format: https://github.com/bitcoin/bitcoin/blob/master/test/functional/test_runner.py#L476
```
drwxr-xr-x   10 rkrux  staff   320B Nov 15 12:25 test_runner_?_🏃_20241115_120930
drwxr-xr-x    7 rkrux  staff   224B Nov 15 15:12 test_runner_?_🏃_20241115_150614
```

Reply:
> Think of each unit test as a separate child class inheriting from the testing setup class and implementing a run() method. Each one constructs a new instance of the setup class, thereby creating a new rand_str.

One thing that I noticed is that there are several such (`1731649167593349000`) time directories when the unit tests are run, each having around one test_name directory inside. Whereas for the functional tests, a dir such as `test_runner_?_🏃_20241115_150614` encapsulates all of them.